src/ansys/tools/visualization_interface/backends/pyvista/widgets/mesh_slider.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides the measure widget for the PyAnsys plotter."""
+from beartype.typing import TYPE_CHECKING
+import pyvista as pv
+
+if TYPE_CHECKING:
+    from ansys.tools.visualization_interface.backends.pyvista.pyvista import Plotter
+
+from pathlib import Path
+
+from vtk import vtkActor, vtkButtonWidget, vtkPNGReader
+
+from ansys.tools.visualization_interface.backends.pyvista.widgets.widget import PlotterWidget
```

Comment:
Can we properly order imports? Seems weird to me this distribution.

Reply:
They are ordered automatically by isort this way, not sure about the logic is following though..

Reply:
It's probably caused by the fact that there is an ``if TYPE_CHECKING`` right in the middle. If you put that one right at the end of the imports, isort will do a better job